.travis.yml

```diff
@@ -4,10 +4,28 @@ script:
   - npm run build
   - npm run test:ci
 node_js:
+  - 6
   - 8
   - 10
   - 12
 
+before_install:
+  - npm config set depth 0
+before_script: >
+  node_version=$(node -v);
+  if [ ${node_version:3:1} = "." ]; then
+    echo "Node 10+"
+    npm install --no-save "eslint@6"
```

Comment:
We don't actually need this line, right? It should already be installed.

Reply:
I think we need the `npm install`, but the rest should be omittable, yes. Removed.